codeworld-server/src/Main.hs

```diff
@@ -20,6 +22,7 @@
 
 module Main where
 
+import qualified "codeworld-compiler" Compile as C
```

Comment:
Same question: what other package defines a `Compile` and requires package imports?